api/index.js

```diff
@@ -19,4 +20,41 @@ var API = (function(userFound) {
   }
 })(user)
 
-module.exports = API
+var DB = (() => {
```

Comment:
Could we create a `/db` folder instead so the imports don't change everywhere or now that you've done the work is that a dumb suggestion?

Reply:
If you think it makes it more maintainable I think it would be worth doing.